styles.css

```
@@ -4200,6 +4200,45 @@ If your plugin does not need CSS, delete this file.
 	background-color: var(--text-error);
 }
 
+/* Non-image attachment preview (PDF, audio, video) */
+.gemini-agent-attachment-file {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	justify-content: center;
+	min-width: 0;
```

Stylelint (17.2.0)

[error] Unexpected duplicate "min-width" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)